gensim/corpora/mmcorpus.py

```diff
@@ -12,21 +12,23 @@
 
 import logging
 
-from gensim import interfaces, matutils
+from gensim.matutils import MmReader, MmWriter
 from gensim.corpora import IndexedCorpus
 
 
 logger = logging.getLogger('gensim.corpora.mmcorpus')
 
 
-class MmCorpus(matutils.MmReader, IndexedCorpus):
+class MmCorpus(MmReader, IndexedCorpus):
     """
     Corpus in the Matrix Market format.
     """
-    def __init__(self, fname):
+    def __init__(self, fname, index_fname=None, transposed=True):
```

Comment:
keep forgetting that you get these requests. you can close/abandon this
particular PR, I'll resubmit once it's fixed up


--Hobson
(503) 974-6274
gh <https://github.com/hobson/> twtr <https://twitter.com/hobsonlane> li
<https://www.linkedin.com/in/hobsonlane> g+
<http://plus.google.com/+HobsonLane/> so
<http://stackoverflow.com/users/623735/hobs>

On Wed, Aug 17, 2016 at 10:17 AM, Lev Konstantinovskiy <
notifications@github.com> wrote:

> In gensim/corpora/mmcorpus.py
> <https://github.com/RaRe-Technologies/gensim/pull/735#discussion_r75165836>
> :
>
> >      """
> >      Corpus in the Matrix Market format.
> >      """
> > -    def __init__(self, fname):
> > +    def __init__(self, fname, index_fname=None, transposed=True):
>
> Thanks! would mak a good separate PR
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RaRe-Technologies/gensim/pull/735/files/4837429dea667f5a2a18b6389436e9bd08ad1d9d..68b20a77b5a369843cf018b0611cc56fe7123af9#r75165836>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAR39wqpcRCEpIkzn7uzA--IjzGORTo3ks5qg0IkgaJpZM4IuMQA>
> .
>